client/src/modules/holidays/holidays.js

```diff
@@ -0,0 +1,105 @@
+angular.module('bhima.controllers')
+.controller('HolidayManagementController', HolidayManagementController);
+
+HolidayManagementController.$inject = [
+  'HolidayService', 'ModalService',
+  'NotifyService', 'uiGridConstants', '$state', 'SessionService',
```

Comment:
Do you use `SessionService`?  It doesn't look like it from here.  Could you remove it?